candid/evm_rpc.did

```diff
@@ -95,7 +95,6 @@ type LogEntry = record {
 };
 type ManageProviderArgs = record {
   providerId : nat64;
-  chainId: opt nat64;
```

Comment:
That seems like a breaking change or am I missing something?

Reply:
This is a breaking change. I've indicated this in the PR title (`feat!`), since any change to mutability could potentially break local workflows and integration tests.